ckcp/test.sh

```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/env bash
 
 #quit if exit status of any cmd is a non-zero value
-set -exuo pipefail
+set -euo pipefail
+# set -x
```

Comment:
Suggestion. Adding a comment above this would be helpful.
#Uncomment the below line to enable debugging.

Reply:
Done